test/export/test_export.py

```diff
@@ -5974,6 +5974,166 @@ def forward(self, x):
 
         self.assertEqual(gm_flat_non_strict(*inp), gm_flat_strict(*inp))
 
+    def test_unflatten_multiple_graphs(self):
```

Comment:
One clarifying question: what would module swapping look like from the user side?

If they don't make distinctions between specialized graphs, it seems like they'd have to manually switch out `n, n@1`, and potentially `p, p@1` if they're also aliasing? Or if they make distinctions for aliasing/specializations then some subset of those.

I don't have context for what swapping looks like today with aliasing - is it one or multiple swaps? - but the main point is, does this strictly introduce more work for swapping, and could we introduce some unflattener API for this?

Reply:
Yeah, they will have to update all `f@i` variants to new `f`. Maybe we should build some convenience APIs to grab these fqn variants.